sdk/core/azure-core/inc/azure/core/internal/tracing/service_tracing.hpp

```diff
@@ -215,6 +215,25 @@ namespace Azure { namespace Core { namespace Tracing { namespace _internal {
                   ->CreateTracer(packageName, packageVersion);
       }
     }
+    /**
+     * @brief Construct a new Tracing Context Factory object
+     *
+     * @param options Client Options for tracing.
+     * @param serviceName Name of the resource provider for the service [See
+     * also](https://docs.microsoft.com/azure/azure-resource-manager/management/azure-services-resource-providers).
+     * @param packageVersion Package version number for the package containing this
+     * service. (https://opentelemetry.io/docs/reference/specification/trace/api/#get-a-tracer).
+     * 
+     * @deprecated This constructor is deprecated and should not be used by any service code. It exists only to support
+     * an earlier release of the Attestation SDK and should not be used by any code beyond that.
+     */
+    [[deprecated]] TracingContextFactory(
```

Comment:
Please also update https://github.com/Azure/azure-sdk-for-cpp/blob/86aa9b4b0cfdef2af103ed2330064079ada68773/doc/DistributedTracing.md?plain=1#L138 to use the 4-parameter version.

Reply:
I"ll be doing that when I make the changelog changes - separate PR.